heffalump/heffalump.go

```diff
@@ -50,8 +58,24 @@ func (h *Heffalump) WriteHell(bw *bufio.Writer) (int64, error) {
 	buf := h.pool.Get().([]byte)
 	defer h.pool.Put(buf)
 
-	if _, err = bw.WriteString("<html>\n<body>\n"); err != nil {
-		return n, err
+	switch cType {
+	case PlainText:
+		if _, err = bw.WriteString("# Chapter 1\n"); err != nil {
+			return n, err
+		}
+		break
```

Comment:
nitpick, just so we can tidy up before merge: no need to add break to any of these cases, switch is only going to hit one path

Reply:
def don't

Reply:
see here: https://go.dev/play/p/xilVd6lOZAj